npm-shrinkwrap.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fh-mbaas-api",
-  "version": "7.0.9",
+  "version": "7.0.10",
```

Comment:
It looks like `7.0.10` is already published, should this be `7.0.11` now as well as in `package.json`?

https://www.npmjs.com/package/fh-mbaas-api